Dependencies/xr/Include/XR.h

```diff
@@ -223,6 +298,7 @@ namespace xr
                 void UpdateAnchor(Anchor&) const;
                 void DeleteAnchor(Anchor&) const;
                 Plane& GetPlaneByID(Plane::Identifier) const;
+                Mesh& GetMeshByID(Mesh::Identifier) const;
```

Comment:
nit: I think we might want to move away from the pattern of having behavior on the frame at some point (having the frame be pure state makes it easier to reason about stuff like threading). This one's of a piece with the ones that are currently here, though, so I think it's not a big deal either way.

Reply:
I'm not sure what the preference will be here. Currently collections for planes, meshes, etc are declared on the XRFrame for WebXR. Is the call out here that we would prefer to have mesh/plane information maintained elsewhere in native code? Or is the suggestion to try and change the WebXR spec?

Reply:
my suggestion would be to keep things this way for now. we can update things here as the WebXR spec changes